src/borg/archiver.py

```diff
@@ -1726,7 +1725,7 @@ def do_debug_dump_manifest(self, args, repository, manifest, key):
 
         data = key.decrypt(None, repository.get(manifest.MANIFEST_ID))
 
-        meta = prepare_dump_dict(msgpack.fallback.unpackb(data, object_hook=StableDict, unicode_errors='surrogateescape'))
+        meta = prepare_dump_dict(msgpack.unpackb(data, object_hook=StableDict, unicode_errors='surrogateescape'))
```

Comment:
does someone remember why this was explicitly using the fallback implementation?